### PR TITLE
Bibcirculation: Add an additional task to the daemon

### PR DESCRIPTION
**Pull request by rsripada**
- Updates the metadata of the records corresponding to all the
  items in the Bibcirc database with the shelf locations.

### CODE REVIEW

File: modules/bibcirculation/lib/bibcirculation_daemon.py [hunk 1]
```diff
@@ -252,17 +261,94 @@ def task_run_core():
...
+            MARC_RECS_STR += "</collection>"
+            timestamp = time.strftime("%Y-%m-%dT%H:%M:%SZ", time.localtime())
+            marcxmlfile = 'MARCxml_booksearch' + '_' + timestamp + '_'
+            fd, marcxmlfile = tempfile.mkstemp(dir=CFG_TMPDIR, prefix=marcxmlfile, suffix='.xml')
+            os.write(fd, MARC_RECS_STR)
+            os.close(fd)
+            write_message("Composed MARCXML saved into %s" % marcxmlfile)
+
+            #Schedule the bibupload task.
+            task_id = task_low_level_submission("bibupload", "BibCirc", "-c", marcxmlfile, "-P", "3")
```
Comment (egabancho) [inline modules/bibcirculation/lib/bibcirculation_daemon.py:331]: do you really need priority 3 here?

**Comment by egabancho** [comment]
Could you please base your branch on latest Invenio master and not cds-ops/prod?
Also it would be nice of you if you can make a pull request to Invenio with the same branch ;-)

File: modules/bibcirculation/lib/bibcirculation_daemon.py (hunk 1 above)
Comment (rsripada) [inline modules/bibcirculation/lib/bibcirculation_daemon.py]: Yes, It's needed to ensure all the uploads are done before midnight of the day of run. This has been discussed and finalized with Sebastian.

**Comment by egabancho** [comment]
#102

**Pull request closed without merging**